recipes/mehari/build.sh

```
@@ -2,5 +2,7 @@
 
 # Make sure bindgen passes on our compiler flags.
 export BINDGEN_EXTRA_CLANG_ARGS="${CPPFLAGS} ${CFLAGS} ${LDFLAGS}"
+export ROCKSDB_LIB_DIR = "/usr/lib/"
```

Shellcheck

[error] Remove spaces around = to assign.

(SC2290)